Lavanya/HTMLassignment2.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+table, th, td {
+  border: 1px solid black;
+}
+</style>
+</head>
+<body>
+
+<h2>First Gen Pokemon Chart</h2>
+
+<table>
+  <col width="130">
+  <col width="80">
+  <col width="80">
+  <col width="80">
+  <tr>
+    <th>Image</th>
+    <th>Name</th> 
+    <th>Type</th>
+    <th>Evolves Into</th>
+  </tr>
+  <tr>
+    <td><img src="http://img4.wikia.nocookie.net/__cb20140328190757/pokemon/images/thumb/2/21/001Bulbasaur.png/200px-001Bulbasaur.png
+      "></td>
+    <td>Bulbasaur</td>
+    <td>Grass/Poison</td>
+    <td><a href="http://pokemon.wikia.com/wiki/Ivysaur">Ivyasaur</a></td>
+  </tr>
+  <tr>
+    <td><img src="	http://img4.wikia.nocookie.net/__cb20140724195345/pokemon/images/thumb/7/73/004Charmander.png/200px-004Charmander.png	http://img4.wikia.nocookie.net/__cb20140724195345/pokemon/images/thumb/7/73/004Charmander.png/200px-004Charmander.png"></td>
+    <td>Charmander</td>
+    <td>Fire</td>
+    <td><a href="http://pokemon.wikia.com/wiki/Charmeleon">Charmeleon</a></td>
+  </tr>
+  <tr>
+    <td><img src="	http://img1.wikia.nocookie.net/__cb20140328191525/pokemon/images/thumb/3/39/007Squirtle.png/200px-007Squirtle.png"></td>
+    <td>Squirtle</td>
+    <td>Water</td>
+    <td><a href="http://pokemon.wikia.com/wiki/Wartortle">Wartortle</a></td>
+  </tr>
+  </table>
+  </body>
+</html>

```

Comment:
Set indentation to tab 4.

Reply:
@LavanyaSirisha This still needs fixing.

Reply:
It gets reset to spaces 4 for some reason.

Reply:
> It gets reset to spaces 4 for some reason.

You need to convert to tabs first. Follow this in the order:

- Press F1
- Convert indentation to tabs
- Press F1
- Indent using tabs
- Press F1
- Format Document

Reply:
Is there no way to change the tab size on the GitHub gist itself?

Reply:
> Is there no way to change the tab size on the GitHub gist itself?

Nothing concrete. Use VS Code, commit and push. That's all.